src/nifs/vanilla/mod.rs

```diff
@@ -379,5 +380,32 @@ impl<C: CurveAffine> VerifyAccumulation<C> for VanillaFS<C> {
     }
 }
 
+/// As part of the vanilla folding scheme, we use the values in the zero instance of the column for
+/// consistency between folding steps
+///
+/// - X0 is
+///     initializing value
+///     or
+///     hash of the state at the end of previous folding step
+/// - X1 is a hash of the state at the end of the current folding step
+pub trait GetConsistencyMarkers<F> {
```

Comment:
can you explain why choose the naming of ConsistencyMarker here? I don't quite get it

Reply:
Of course

X0, X1 perform the function of checking the consistency of the previous and the next step. I.e. we check the consistency of step circuits between folding steps.